.github/workflows/tests.yml

```diff
@@ -44,4 +44,3 @@ jobs:
 env:
   CARGO_TERM_COLOR: always
   CARGO_INCREMENTAL: 0
-  RUSTC_WRAPPER: sccache
```

Comment:
Probably you should delete `sccache` in `cache-util restore/save ...` as well? It seems doesn't make sense anymore.